hebiros_advanced_examples/CMakeLists.txt

```diff
@@ -19,17 +19,34 @@ add_executable(example_rviz_node src/rviz/example_rviz_node.cpp)
 add_executable(example_moveit_node src/moveit/example_moveit_node.cpp)
 add_executable(example_gazebo_node src/gazebo/example_gazebo_node.cpp)
 add_executable(example_mobile_robot_node src/mobile_robot/example_mobile_robot_node.cpp)
+add_executable(example_omnibase_free_roam src/omnibase/example_omnibase_free_roam.cpp)
+add_executable(example_omnibase_odometry src/omnibase/example_omnibase_odometry.cpp)
+
 
 add_dependencies(example_set_gains_node ${catkin_EXPORTED_TARGETS})
 add_dependencies(example_rviz_node ${catkin_EXPORTED_TARGETS})
 add_dependencies(example_moveit_node ${catkin_EXPORTED_TARGETS})
 add_dependencies(example_gazebo_node ${catkin_EXPORTED_TARGETS})
 add_dependencies(example_mobile_robot_node ${catkin_EXPORTED_TARGETS})
+add_dependencies(example_omnibase_free_roam ${catkin_EXPORTED_TARGETS})
+add_dependencies(example_omnibase_odometry ${catkin_EXPORTED_TARGETS})
+
 
 target_link_libraries(example_set_gains_node ${catkin_LIBRARIES})
 target_link_libraries(example_rviz_node ${catkin_LIBRARIES})
 target_link_libraries(example_moveit_node ${catkin_LIBRARIES})
 target_link_libraries(example_gazebo_node ${catkin_LIBRARIES})
 target_link_libraries(example_mobile_robot_node ${catkin_LIBRARIES})
+target_link_libraries(example_omnibase_free_roam ${catkin_LIBRARIES})
+target_link_libraries(example_omnibase_odometry ${catkin_LIBRARIES})
 
+install(PROGRAMS
+  src/omnibase/key_read.py
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
 
+install(FILES
```

Comment:
:+1: this reminds me, we should make sure the "install" target works for other example launch files, too.